### PR TITLE
add data_checksums option to initdb

### PR DESCRIPTION
**Pull request by tjikkun**

### CODE REVIEW

File: manifests/server/initdb.pp [hunk 1]
```diff
@@ -82,11 +83,17 @@
       $require_before_initdb = [$datadir]
     }
 
-    $initdb_command = $locale ? {
+    $ic_locale = $locale ? {
```
Comment (tphoney) [inline manifests/server/initdb.pp]: what is the meaning of this variable name, it is a little terse. was there any reason to rename it ?

**Comment by tphoney** [comment]
Looks pretty good otherwise.

File: manifests/server/initdb.pp (hunk 1 above)
Comment (tjikkun) [inline manifests/server/initdb.pp]: Well, it was like this:
```
$ic_base = "${initdb_path} --encoding '${encoding}' --pgdata '${datadir}'"
$ic_xlog = $ic_base + optional xlogdir
$initdb_command = $ic_xlog + optional locale
```
Now I changed it so:
```
$ic_base = "${initdb_path} --encoding '${encoding}' --pgdata '${datadir}'"
$ic_xlog = $ic_base + optional xlogdir
$ic_locale = $ic_xlog + optional locale
$initdb_command = $ic_locale + optional data-checksums
```
Basically ic_ (probably short for initdb_command) was there before, I just kept the same convention.

**Comment by tphoney** [comment]
thanks for the work @tjikkun :)

**Pull request merged**